### PR TITLE
Parsers for 389 Directory Server access and errors logs for 1.x branch

### PR DESCRIPTION
**Pull request by PaulWay**
See https://github.com/RedHatInsights/insights-core/pull/116 for the main work - this PR is for the 1.x branch.

### CODE REVIEW

File: insights/config/specs.py [hunk 1]
```diff
@@ -58,6 +58,8 @@
     "df_-li"                    : CommandSpec("/bin/df -li"),
     "dig"                       : CommandSpec("/usr/bin/dig +dnssec . DNSKEY"),
     "dirsrv"                    : SimpleFileSpec("etc/sysconfig/dirsrv"),
+    "dirsrv_logs"               : PatternSpec("var/lib/dirsrv/.*/access"),
+    "dirsrv_errs"               : PatternSpec("var/lib/dirsrv/.*/errors"),
```
Comment (bfahr) [inline insights/config/specs.py]: @PaulWay would you mind to swap the order of these two lines?

File: insights/parsers/dirsrv_logs.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        >>> for access_log in shared[DirSrvAccessLog]:
+        ...     print "Path:", access_log.path
+        ...     connects = access_log.get('connection from')
+        ...     print "Connection lines:", len(connects)
+        ...
+        Path: /var/log/dirsrv/slapd-EXAMPLE-COM/access
+        Connection lines: 2
+    '''
+    pass
+    # def get_after(self, timestamp, lines=None):
```
Comment (PaulWay) [inline insights/parsers/dirsrv_logs.py]: I'd really rather leave them there until the LogFileOutput get_after work in #118 and #119 gets merged, then I'll update these and remove the comments.  I commented the code out just to test it against the branches without that addition.

**Comment by bfahr** [comment]
@PaulWay @xiangce the simple resolution to the extra comments is to wait until #119 is merged and then make the additional changes to this PR.

**Comment by PaulWay** [comment]
OK, I don't know what's gone wrong with this but the history has been adgered; I'm going to close this and redo a new one.

**Pull request closed without merging**